src/Type.zig

```diff
@@ -1603,6 +1603,9 @@ fn abiSizeInnerOptional(
 }
 
 pub fn ptrAbiAlignment(target: *const Target) Alignment {
```

Comment:
Seems like this should move to `target.zig` since it has nothing to do with `Type`. But that doesn't necessarily have to happen in this PR.